lib/ansible/modules/network/f5/bigip_pool_member.py

```diff
@@ -384,7 +402,8 @@ def main():
         description=dict(type='str'),
         rate_limit=dict(type='int'),
         ratio=dict(type='int'),
-        preserve_node=dict(type='bool', default=False)
+        preserve_node=dict(type='bool', default=False),
+        priority_group=dict(type='int')
```

Comment:
@ossark Thanks for the PR, just one change then I can get this merged. Thanks :)